src/aggregator/client/conn_options.go

```diff
@@ -32,7 +32,7 @@ import (
 const (
 	defaultConnectionTimeout            = 2 * time.Second
 	defaultConnectionKeepAlive          = true
-	defaultWriteTimeout                 = 100 * time.Millisecond
+	defaultWriteTimeout                 = time.Duration(0)
```

Comment:
This defaults to no timeout right?

Reply:
Correct yes, we have TCP Keep alive so we don't really need timeout (since the connection will get broken in 10s or so by the keep alive if it's actually stale/dead and that will timeout the write that way).